src/main/webapp/view/transcript/transcript.js

```diff
@@ -142,3 +142,13 @@ function removeBold(transcriptLineLiElement) {
   transcriptLineLiElement.classList.add(DEFAULT_FONT_WEIGHT);
   transcriptLineLiElement.classList.remove(BOLD_FONT_WEIGHT);
 }
+
+/**
+ * Scrolls `transcriptLine` to the top of
+ * `TRANSCRIPT_CONTAINER`.
```

Comment:
It's unclear what `TRANSCRIPT_CONTAINER` is without reading other code.  Change to "Scrolls `transcriptLine` to the top of the transcript area."?

Reply:
Gotcha, updated!